3-enrich/stream-enrich/build.sbt

```diff
@@ -10,47 +10,93 @@
+  // Test
+  Dependencies.Libraries.specs2,
+  Dependencies.Libraries.scalacheck
+)
+
+lazy val buildSettings = Seq(
+  organization  :=  "com.snowplowanalytics",
+  name          :=  "snowplow-stream-enrich",
+  version       :=  "0.15.0-rc1",
+  description   :=  "The streaming Snowplow Enrichment process",
+  scalaVersion  :=  "2.11.11",
+  scalacOptions :=  BuildSettings.compilerOptions,
+  scalacOptions in (Compile, console) ~= { _.filterNot(Set("-Ywarn-unused-import")) },
+  scalacOptions in (Test, console)    := (scalacOptions in (Compile, console)).value,
+  javacOptions  :=  BuildSettings.javaCompilerOptions,
+  resolvers     ++= Dependencies.resolutionRepos
+)
+
+lazy val allSettings = buildSettings ++
+  BuildSettings.sbtAssemblySettings ++
+  Seq(libraryDependencies ++= commonDependencies)
+
 lazy val root = project.in(file("."))
+  .settings(buildSettings)
+  .aggregate(core, kinesis, pubsub, kafka, nsq, stdin)
+
+lazy val core = project
+  .settings(moduleName := "snowplow-stream-enrich")
+  .settings(buildSettings)
+  .settings(libraryDependencies ++= commonDependencies)
+  .enablePlugins(BuildInfoPlugin)
   .settings(
-    organization  :=  "com.snowplowanalytics",
-    name          :=  "snowplow-stream-enrich",
-    version       :=  "0.14.0",
-    description   :=  "The Snowplow Enrichment process, implemented as an Amazon Kinesis app",
-    scalaVersion  :=  "2.11.11",
-    scalacOptions :=  BuildSettings.compilerOptions,
-    scalacOptions in (Compile, console) ~= { _.filterNot(Set("-Ywarn-unused-import")) },
-    scalacOptions in (Test, console)    := (scalacOptions in (Compile, console)).value,
-    javacOptions  :=  BuildSettings.javaCompilerOptions,
-    resolvers     ++= Dependencies.resolutionRepos,
-    shellPrompt   := { _ => "stream-enrich> " }
-  )
-  .settings(BuildSettings.scalifySettings)
-  .settings(BuildSettings.sbtAssemblySettings)
-  .settings(
-    libraryDependencies ++= Seq(
-      // Java
-      Dependencies.Libraries.kinesisSdk,
-      Dependencies.Libraries.dynamodbSdk,
-      Dependencies.Libraries.s3Sdk,
-      Dependencies.Libraries.kinesisClient,
-      Dependencies.Libraries.kafkaClients,
-      Dependencies.Libraries.commonsCodec,
-      Dependencies.Libraries.config,
-      Dependencies.Libraries.slf4j,
-      Dependencies.Libraries.log4jOverSlf4j,
-      Dependencies.Libraries.nsqClient,
-      // Scala
-      Dependencies.Libraries.scopt,
-      Dependencies.Libraries.scalaz7,
-      Dependencies.Libraries.json4s,
-      Dependencies.Libraries.json4sJackson,
-      Dependencies.Libraries.pureconfig,
-      Dependencies.Libraries.snowplowRawEvent,
-      Dependencies.Libraries.snowplowCommonEnrich,
-      Dependencies.Libraries.igluClient,
-      Dependencies.Libraries.snowplowTracker,
-      // Test
-      Dependencies.Libraries.specs2,
-      Dependencies.Libraries.scalacheck
-    )
+    buildInfoKeys := Seq[BuildInfoKey](organization, name, version,
```

Comment:
Didn't know about this plugin. Do you think we should standardize on it for all code bases? (instead of custom `scalify` settings)

Reply:
yes there are a few tickets already open: https://github.com/snowplow/snowplow/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+buildinfo